bokeh/models/tools.py

```diff
@@ -210,6 +211,31 @@ class CrosshairTool(Tool):
     only a vertical line will be drawn.
     """)
 
+    line_color = Color(default="black", help="""
+    A color to use to stroke paths with.
+
+    Acceptable values are:
+
+    - any of the 147 named `CSS colors`_, e.g ``'green'``, ``'indigo'``
+    - an RGB(A) hex value, e.g., ``'#FF0000'``, ``'#44444444'``
+    - a 3-tuple of integers (r,g,b) between 0 and 255
+    - a 4-tuple of (r,g,b,a) where r,g,b are integers between 0..255 and a is between 0..1
+
+    .. _CSS colors: http://www.w3schools.com/cssref/css_colornames.asp
+
+    """)
+
+    line_width = Float(default=1, help="""
+    Stroke width in units of pixels.
+    """)
+
+    line_alpha = Float(default=1.0, help="""
+    An alpha value to use to stroke paths with.
+
+    Acceptable values are floating point numbers between 0 (transparent)
+    and 1 (opaque).
+
+    """)
```

Comment:
I personally prefer being explicit, and not using `LineProps` (since they cannot all be supported). @mattpap do you still think `LineProps` is better, given that some of them will be non-functional in some cases?